src/layers/core/hexagon-layer/hexagon-layer.js

```diff
@@ -217,7 +218,7 @@ export default class HexagonLayer extends CompositeLayer {
     // viewport props
     const {positionOrigin, projectionMode, modelMatrix} = this.props;
 
-    return new HexagonCellLayer({
+    return {
```

Comment:
Add a comment saying `// return a bunch of props to the sublayer constructor specified in  getSubLayer()`